# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1385091481 18000
#      Thu Nov 21 22:38:01 2013 -0500
# Node ID 7c48493d98e14e3997acc6066e553f1eada92385
# Parent  96a4ac0dc01f72219a8a5f748fc72054ffd9a50c
Use currentDateTime instead of currentDateTimeUtc for Qt compatibility

* main-window.cc (main_window::main_window, news_reader::process) Use
QDateTime::currentDateTime instead of QDateTime::currentDateTimeUtc for
compatibility with Qt 4.6.

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -102,17 +102,17 @@ main_window::main_window (QWidget *p)
         = settings->value ("news/allow_web_connection", true).toBool ();
 
       last_checked
         = settings->value ("news/last_time_checked", QDateTime ()).toDateTime ();
 
       serial = settings->value ("news/last_news_item", 0).toInt ();
     }
 
-  QDateTime current = QDateTime::currentDateTimeUtc ();
+  QDateTime current = QDateTime::currentDateTime ();
   QDateTime one_day_ago = current.addDays (-1);
 
   if (connect_to_web
       && (! last_checked.isValid () || one_day_ago > last_checked))
     load_and_display_community_news (serial);
 
   // We have to set up all our windows, before we finally launch octave.
   construct ();
@@ -373,17 +373,17 @@ news_reader::process (void)
         {
           if (serial >= 0)
             {
               QSettings *settings = resource_manager::get_settings ();
 
               if (settings)
                 {
                   settings->setValue ("news/last_time_checked",
-                                      QDateTime::currentDateTimeUtc ());
+                                      QDateTime::currentDateTime ());
 
                   settings->sync ();
                 }
 
               QString tag ("community-news-page-serial=");
 
               int b = html_text.indexOf (tag);
 
